src/lib/import-export/import/handlers/backup-handler-zip.ts

```diff
@@ -16,13 +16,18 @@ export class BackupHandlerZip extends EventEmitter implements BackupHandler {
 	async extractFiles( file: BackupArchiveInfo, extractionDirectory: string ): Promise< void > {
 		this.emit( ImportEvents.BACKUP_EXTRACT_START );
 		return new Promise( ( resolve, reject ) => {
-			this.emit( ImportEvents.BACKUP_EXTRACT_PROGRESS );
+			this.emit( ImportEvents.BACKUP_EXTRACT_PROGRESS, {
+				progress: 0,
+			} as BackupExtractProgressEventData );
 			const zip = new AdmZip( file.path );
 			zip.extractAllToAsync( extractionDirectory, true, undefined, ( error?: Error ) => {
 				if ( error ) {
 					this.emit( ImportEvents.BACKUP_EXTRACT_ERROR, { error } );
 					reject( error );
 				}
+				this.emit( ImportEvents.BACKUP_EXTRACT_PROGRESS, {
+					progress: 1,
+				} as BackupExtractProgressEventData );
```

Comment:
After we introduce changes related to https://github.com/Automattic/dotcom-forge/issues/8329, we should review this part in case there are listeners we can use to report the progress.